src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -2201,4 +2201,47 @@ private class ACCT_IndividualAccounts_TEST {
         System.assertEquals(tempUser.Id, accAfterUpdate.ownerId);
     }
 
+    /*****************************************************************************************************************************************
+    * @description Helper method to set Default Account Model and Administrative Account Record Type.
+    */
+
+    private static void setAdminAccRecTypeAsBizOrgHelper() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getBizAccRecTypeID(),
+                                                         Administrative_Account_Record_Type__c =  UTIL_Describe_API.getBizAccRecTypeID()));
+    }
+
+    /***********************************************************************************************************************************************
+    * @description Test to ensure Account owner update changes owner on Account when Administrative Account Record Type  and Account Model are
+    * Business Org Record Type
+    */
+    @isTest
+    private static void testContactOwnerUpdateWithBizAccRecTypeAsAdminAccRecType() {
+
+        ACCT_IndividualAccounts_Test.setAdminAccRecTypeAsBizOrgHelper();
+
+        // Insert a user
+        Id sysAdminProfileId = UTIL_Profile.getInstance().getProfileIds(UTIL_Profile.SYSTEM_ADMINISTRATOR)[0];
+        User tempUser = new User(LastName = 'TestSysAdminUserA', Email = 'test_insertAsOtherUser@email.npsp',
+                                       ProfileId = sysAdminProfileId, isActive = true, UserName = 'test_insertAsOtherUser@email.npsp',
+                                     Alias = 'tu08534', TimeZoneSidKey = 'America/Los_Angeles',
+                                    LocaleSidKey = 'en_US', LanguageLocaleKey = 'en_US', EmailEncodingKey = 'ISO-8859-1');
+        insert tempUser;
+
+        Contact con = UTIL_UnitTestData_API.getContact();
+        con.LastName = 'test1234';
+        insert con;
+
+        Test.startTest();
+            con.OwnerId = tempUser.Id;
+            update con;
+        Test.stopTest();
```

Comment:
One of the things to note about `Test.startTest()` and `Test.stopTest()` is that their ideal use case is to create a separate set of limits for tests. As such, we should use this only to call methods or use dml operations that we are testing. This allows for isolated profiling when we need to perform it.

```suggestion
        con.OwnerId = tempUser.Id;
        
        Test.startTest();
            update con;
        Test.stopTest();
```